Homework3/Task04/task04.py

```diff
@@ -0,0 +1,5 @@
+def is_armstrong(number: int) -> bool:
+    digits = list(map(int, str(number)))
```

Comment:
why do you need a list here?

Reply:
I need list() here because map() returns a map object that has no attribute such as len or sum that I need.

Reply:
`sum` is not an attribute. You don't use this list with a sum.